meson_options.txt

```diff
@@ -2,6 +2,7 @@ option('glibcxx_asserts', type : 'boolean', value : false)
 option('use_system_vulkan', type : 'feature', value : 'disabled', description: 'Use system vulkan headers instead of the provided ones')
 option('use_system_spdlog', type : 'feature', value : 'disabled', description: 'Use system spdlog library')
 option('vulkan_datadir', type : 'string', value : '', description: 'Path to the system vulkan headers data directory if different from MangoHud\'s datadir')
+option('bashcompletiondir', type: 'string', description: 'Directory for bash completion script ["no" disables]')
```

Comment:
IMHO the extra toggle is an overkill. The completion files are not even 50 lines in total, so I'd just install them always.

Reply:
This option specifies _where_ to put the bash completions.
It may not be always in `${prefix}/share/bash-completion`.
It just happens to allow disabling it.
I copied it from systemd's meson config.

Reply:
The existing logic will work with virtually any linux distro, even ones like Yocto. So in practical terms, the toggle is used for disabling the lot.

The fragment looked oddly familiar - thanks for confirming it's systemd origin.